src/orchard/java.clj

```diff
@@ -323,28 +323,43 @@
          (filter identity)
          (distinct))))
 
+(defn trim-one-dot
```

Comment:
Probably this shouldn't be public API. I'm also not sure why exactly do you need this. Remove `.` from constructors?

Reply:
Yep, it was already there, I have just factored it out. I will make it private.